Source/Core/VideoBackends/Software/Rasterizer.cpp

```diff
@@ -28,14 +28,14 @@ static constexpr int BLOCK_SIZE = 2;
 struct SlopeContext
 {
   SlopeContext(const OutputVertexData* v0, const OutputVertexData* v1, const OutputVertexData* v2,
-               s32 x0, s32 y0, s32 x_off, s32 y_off)
-      : x0(x0), y0(y0)
+               s32 _x0, s32 _y0, s32 x_off, s32 y_off)
```

Comment:
Generally we place the underscore on the other size like `x0_` ditto for other occurrences